flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/inference/strategies/ArrayElementOutputTypeStrategy.java

```diff
@@ -24,24 +24,19 @@
 import org.apache.flink.table.types.DataType;
 import org.apache.flink.table.types.inference.CallContext;
 import org.apache.flink.table.types.inference.TypeStrategy;
-import org.apache.flink.table.types.logical.LogicalTypeRoot;
 
 import java.util.Optional;
 
-/** Specific {@link TypeStrategy} for {@link BuiltInFunctionDefinitions#ARRAY_MAX}. */
+/**
```

Comment:
Let's rebase that on top of https://github.com/apache/flink/pull/22917

Reply:
ok